client/ayon_kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -62,16 +73,37 @@ def process(self, context):
                 "family_requirements"
             ]
 
+        farm_status_change=False
+
+        families= []
+        for instance in context:
+                families += set(
+                [instance.data["family"]] + instance.data.get("families", [])
+            )
+        self.log.debug(f'Falimies in context {families}')
```

Comment:
I've tried to do this aproach and it may create a mess since it can create a ton of farm statuses as multiple instances can be rendering on farm. And the logic here is like that: at each publish something should be posted as a review if we dont have anything for the review so it renders on farm since farm families in not consistent across apps, nuke have farm family huildini will have only DEBUG: ['redshift_rop', 'workfile'] or something like, I know its issue if I dont have anything that can be rendered on farm or for review we will have wrong status with this problem i thought you will help me